meta-balena-common/recipes-bsp/grub/grub-efi_%.bbappend

```diff
@@ -9,5 +20,19 @@ do_deploy:append:class-target() {
     install -d ${DEPLOYDIR}/grub/${GRUB_TARGET}-efi/
 }
 
-# build in additional required modules
-GRUB_BUILDIN:append = " regexp probe gzio"
+do_mkimage() {
+    cd ${B}
+
+    if [ -f "${DEPLOYDIR}/grub.gpg" ]; then
+        GRUB_PUBKEY_ARG="--pubkey ${DEPLOYDIR}/grub.gpg"
+    fi
+
+    # Search for the grub.cfg on the local boot media by using the
+    # built in cfg file provided via this recipe
+    grub-mkimage -c ../cfg -p ${EFIDIR} -d ./grub-core/ \
+           -O ${GRUB_TARGET}-efi -o ./${GRUB_IMAGE_PREFIX}${GRUB_IMAGE} \
+           ${GRUB_BUILDIN} ${GRUB_PUBKEY_ARG}
+}
```

Comment:
This needs
```
do_mkimage[depends] += " \
    balena-keys:do_deploy \
    "
```

Reply:
Right, I was thinking this recipe would inherit the class that introduced the dependency but now we removed that.